crates/test/src/lib.rs

```diff
@@ -59,3 +68,9 @@ macro_rules! wasm_bindgen_test_configure {
 
 #[path = "rt/mod.rs"]
 pub mod __rt;
+
+// Make this only available to wasm32 so that we don't
+// import minicov on other archs.
+// That way you can use normal cargo test without minicov
```

Comment:
I agree this is a good point, but then we should move the dependency to `[target.'cfg(target_arch = "wasm32")'.dependencies]` as well.